code/modules/mob/living/carbon/xenomorph/strains/castes/facehugger/watcher.dm

```diff
@@ -21,5 +21,5 @@
 	name = "Watcher Facehugger Behavior Delegate"
 
 /datum/behavior_delegate/facehugger_watcher/on_life()
-	if(bound_xeno.body_position == STANDING_UP && !(locate(/obj/effect/alien/weeds) in get_turf(bound_xeno)))
+	if(bound_xeno.body_position == STANDING_UP && !(locate(/obj/effect/alien/weeds) in get_turf(bound_xeno)) && !bound_xeno.is_zoomed)
 		bound_xeno.adjustBruteLoss(1)
```

Comment:
```suggestion
	// Sap health if we're standing, not on weeds, and not zoomed out
	if(bound_xeno.body_position != STANDING_UP)
		return
	if(bound_xeno.is_zoomed)
		return
	if(locate(/obj/effect/alien/weeds) in get_turf(bound_xeno))
		return
	bound_xeno.adjustBruteLoss(1)
```
This check has become unwieldly. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-early-returns Also its slightly better to check easy checks before more complicated check, so do the is_zoomed check earlier.